collect_app/src/main/java/org/odk/collect/android/widgets/RangeWidget.java

```diff
@@ -172,14 +170,10 @@ private void setUpSeekBar() {
         seekBar.setOnSeekBarChangeListener(this);
         if (isRTL()) {
             float rotate = seekBar.getRotation();
-            if (Build.VERSION.SDK_INT == Build.VERSION_CODES.KITKAT ||
-                    Build.VERSION.SDK_INT == Build.VERSION_CODES.JELLY_BEAN_MR2) {
+            if (Build.VERSION.SDK_INT == Build.VERSION_CODES.KITKAT) {
```

Comment:
There is something wrong here, `KITKAT` won't be supported so you don't need this check.